Pipfile

```diff
@@ -10,7 +10,7 @@ python_version = "3"
 #
 # NuCypher
 #
-umbral = "*"
+umbral = "==0.1.1a3"
```

Comment:
This is already pinned in `master`. Shouldn't this be detected by GH as a merge conflict? BTW, perhaps this PR should be rebased on top of `master`.

Reply:
Dropped and rebased. :heavy_check_mark: